internal/pkg/specs/achievement.go

```diff
@@ -74,3 +74,8 @@ func (req *UpdateAchievementRequest) Validate() error {
 
 	return nil
 }
+
+type DeleteAchievementRequest struct {
```

Comment:
No need to define struct for deletion oration, 

Service method signature like : 

`DeleteAchievement(ctx context.Context, profileID, achievementID int)`


same for all

Reply:
done